packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -56,11 +66,26 @@
                     : undefined,
         },
     ]
-
-    let modal: Modal
-    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
     $: $selectedAccountVestingPayouts, (timeUntilNextPayout = getTimeUntilNextPayout())
 
+    onMount(() => {
+        getMinRequiredStorageDeposit()
+        areOutputsReadyForConsolidation()
```

Comment:
I think we should make `areOutputsReadyForConsolidation` reactive on wallet changes, if you receive a new output or something it should automatically update